contents/python/introduction-to-python.md

```diff
@@ -48,4 +48,13 @@ To get started, take a look at:
 *   [matplotlib Tutorials](http://matplotlib.org/users/tutorials.html#introductory)
 *   [scikit-learn Basics](http://scikit-learn.org/stable/tutorial/basic/tutorial.html)
 *   [scikit-learn Machine Learning tutorials](http://scikit-learn.org/stable/tutorial/index.html) are also available for those looking to pick up machine learning.
-    *   If you learn best by working on projects, fret not! [Kaggle](http://kaggle.com) is a treasure trove of real-world datasets that you can hone your data science skills on.
\ No newline at end of file
+*   If you learn best by working on projects, fret not! [Kaggle](http://kaggle.com) is a treasure trove of real-world datasets that you can hone your data science skills on.
+
+## Virtual Environment
+
+When starting out Python projects, you will be recommended to install dependencies using [virtual environment](https://docs.python.org/3/tutorial/venv.html) to remove the possibility of dependencies conflict. This is especially a good practice when managing dependencies from different projects which might have different python versions.
```

Comment:
If this instruction is meant for starting out, shouldn't it be directly under `An Introduction to Python`?

Would `different projects which` ~might have~ `relies on different python versions and packages` sounds better?

Reply:
That sounds good. I do believe having it under `Getting Started With Python` is more appropriate. Thanks for the advice